blockchain/indexers/cfindex.go

```diff
@@ -218,7 +215,7 @@ func (idx *CfIndex) ConnectBlock(dbTx database.Tx, block *btcutil.Block,
 		return err
 	}
 
-	if err := storeFilter(dbTx, block, f, false); err != nil {
+	if err := storeFilter(dbTx, block, f, 0); err != nil {
```

Comment:
Similar comment here as to the other repo (`btcutil`): we should be using an enum here and also above in the function signature. So anywhere that a `uint8` is currently passed, should be morphed to instead pass `FilterType` (or w/e the variable is called).